Feature/framwework structure

antho1404 opened the pull request.

dependabot-preview[bot] committed:
Bump vuepress-plugin-google-tag-manager from 0.0.3 to 0.0.4

Bumps [vuepress-plugin-google-tag-manager](https://github.com/bykclk/vuepress-plugin-google-tag-manager) from 0.0.3 to 0.0.4.
- [Release notes](https://github.com/bykclk/vuepress-plugin-google-tag-manager/releases)
- [Commits](https://github.com/bykclk/vuepress-plugin-google-tag-manager/commits)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

NicolasMahe committed:
Improve example of process task reference

NicolasMahe committed:
Improve example of process task reference

NicolasMahe committed:
Update process filter with ref and predicate

NicolasMahe committed:
Update process filter with ref and predicate

NicolasMahe committed:
fix process filter predicate contains type

NicolasMahe committed:
fix process filter predicate contains type

antho1404 committed:
documentation reorganization

antho1404 committed:
documentation reorganization

NicolasMahe commented on .vuepress/config.js:
```diff
@@ -0,0 +1,104 @@
+            text: 'Libraries',
+            items: [
+              { text: 'NodeJS', link: 'https://github.com/mesg-foundation/js-sdk' },
+              { text: 'Golang (Service)', link: 'https://godoc.org/github.com/mesg-foundation/engine/client/service' },
```
doesn't exist anymore

NicolasMahe commented on .vuepress/config.js:
```diff
@@ -0,0 +1,104 @@
+            items: [
+              { text: 'NodeJS', link: 'https://github.com/mesg-foundation/js-sdk' },
+              { text: 'Golang (Service)', link: 'https://godoc.org/github.com/mesg-foundation/engine/client/service' },
+              { text: 'C# (Not official)', link: 'https://github.com/neville-nazerane/mesg-apis' }
```
not updated since 1 year

NicolasMahe commented on .vuepress/config.js:
```diff
@@ -0,0 +1,104 @@
+            ]
+          },
+          {
+            text: 'Libraries',
```
should we just remove this links?

antho1404 committed:
remove libraries section

antho1404 committed:
remove libraries section

williamzmorgan committed:
Update README.md

Added and removed a bit of text here for organization and ease of understanding.

williamzmorgan committed:
Update README.md

Added and removed a bit of text here for organization and ease of understanding.

dependabot-preview[bot] committed:
Bump vuepress from 1.4.0 to 1.4.1

Bumps [vuepress](https://github.com/vuejs/vuepress/tree/HEAD/packages/vuepress) from 1.4.0 to 1.4.1.
- [Release notes](https://github.com/vuejs/vuepress/releases)
- [Changelog](https://github.com/vuejs/vuepress/blob/master/CHANGELOG.md)
- [Commits](https://github.com/vuejs/vuepress/commits/v1.4.1/packages/vuepress)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
Bump @vuepress/plugin-google-analytics from 1.4.0 to 1.4.1

Bumps [@vuepress/plugin-google-analytics](https://github.com/vuejs/vuepress/tree/HEAD/packages/@vuepress/plugin-google-analytics) from 1.4.0 to 1.4.1.
- [Release notes](https://github.com/vuejs/vuepress/releases)
- [Changelog](https://github.com/vuejs/vuepress/blob/master/CHANGELOG.md)
- [Commits](https://github.com/vuejs/vuepress/commits/v1.4.1/packages/@vuepress/plugin-google-analytics)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

antho1404 committed:
Merge pull request #122 from mesg-foundation/dependabot/npm_and_yarn/vuepress-1.4.1

Bump vuepress from 1.4.0 to 1.4.1

antho1404 committed:
Merge pull request #117 from mesg-foundation/dependabot/npm_and_yarn/vuepress-plugin-google-tag-manager-0.0.4

Bump vuepress-plugin-google-tag-manager from 0.0.3 to 0.0.4

antho1404 committed:
Merge pull request #123 from mesg-foundation/dependabot/npm_and_yarn/vuepress/plugin-google-analytics-1.4.1

Bump @vuepress/plugin-google-analytics from 1.4.0 to 1.4.1

antho1404 committed:
remove generated doc and cleanup

antho1404 committed:
Update menu

antho1404 committed:
cleanup

antho1404 committed:
Merge branch 'feature/framwework-structure' of https://github.com/mesg-foundation/docs into feature/framwework-structure

antho1404 committed:
fix wrong merge

antho1404 committed:
add missing css

antho1404 commented:
closing in favor of https://github.com/liteflow-labs/docs/pull/1
